ios/RNRadar.m

```diff
@@ -357,7 +357,7 @@ - (void)didUpdateLocationPermissionStatus:(RadarLocationPermissionStatus *)statu
             NSMutableDictionary *dict = [NSMutableDictionary new];
             [dict setObject:[Radar stringForStatus:status] forKey:@"status"];
             if (token != nil) {
-                [dict setObject:[token dictionaryValue] forKey:@"token"];
+                [dict setObject:[RNRadar mapLocationPermissionStatus:[token dictionaryValue]] forKey:@"token"];
```

Comment:
can you add a comment explaining this line, its not evident why we are doing this at first glance

Reply:
ios returns the enum as SnakeCaseString, android returns it as CAP_WITH_UNDERSCORE, just making them the same for react-native. I'll add a comment in code.

Reply:
this function is at the top of the file.